### PR TITLE
updated desgin pages and its css

### PR DESCRIPTION
**Pull request by bharti3bk**

### CODE REVIEW

**Comment by bharti3bk** [comment]
image

File: great-idea/services.html [hunk 1]
```diff
@@ -16,91 +16,114 @@
...
-
-Copyright Great Idea! 2018
+  <nav>
+    <a href="#">Services</a>
+    <a href="#">Product</a>
+    <a href="#">Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+    <a href="#"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
```
Comment (alasalle) [inline great-idea/services.html]: The logo anchor should point to your index page.

File: great-idea/services.html [hunk 2]
```diff
@@ -16,91 +16,114 @@
...
+  </nav>
+
+  <header>
+    <div><img class="services-header-img" src="img/services-header.jpg" alt="Our services header image"></div>
+  
+    <div>
```
Comment (alasalle) [inline great-idea/services.html:32]: Just a thought for the future: some of these divs could be sections instead.

**Review by alasalle — COMMENTED** [review]
Hey, Fnu, this is a solid effort. Your styling is top-notch. Your code for the index.html page seems to be missing. Also, the title of the services.html page (in the head part) should be Great Idea! - Services.

**Pull request merged**